Njord/src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -23,15 +38,56 @@ public class ElevatorSubsystem extends SubsystemBase {
   public enum ElevatorState {
     S_GoingUp, S_GoingDown, S_Stopped
     }
+
+  private final MutVoltage mVoltage = Volts.mutable(0);
+  private final MutDistance mDistance = Meters.mutable(0);
+  private final MutLinearVelocity mVelocity = MetersPerSecond.mutable(0);
+
+private Voltage setVolts = (Voltage volts) -> setVoltage(volts);
+private double setLog = (SysIdRoutineLog) -> setLog(SysIdRoutineLog);
+
+
+
+private final SysIdRoutine mSysIdRoutine = 
+   new SysIdRoutine(
+     new SysIdRoutine.Config(Volts.per(Seconds).of(1.2), Volts.of(12), Seconds.of(10)),
+     new SysIdRoutine.Mechanism(
+      voltage -> {
+        elevatorMotor1.setVoltage(voltage);
+      },
+        log -> {
+        log.motor("drive")
+        .voltage(
+         mVoltage.mut_replace(
+           elevatorMotor1.get() * RobotController.getBatteryVoltage(), Volts)
+          .linearPosition(mDistance.mut_replace(elevatorMotor1.getPosition().getValueAsDouble(), Meters)
+          .linearVelocity(
+            m_velocity.mut_replace(elevatorMotor1.getPosition().getRate(), MetersPerSecond))));
```

Comment:
This is probably not meters per second either. We'll have to figure this out.

Reply:
m_velocity is not a thing. Copy and paste error? 

Which var has our velocity? We might not need all this extra code.

Reply:
getRate is not something from getPosition. Seems like we need a rate or speed call from the talon.

Reply:
> m_velocity is not a thing. Copy and paste error?
> 
> Which var has our velocity? We might not need all this extra code.

m_velocity should be mVelocity

```suggestion
            mVelocity.mut_replace(elevatorMotor1.getPosition().getRate(), MetersPerSecond))));
```

Reply:
> getRate is not something from getPosition. Seems like we need a rate or speed call from the talon.

I think we want getVelocity. But getVelocity is angular velocity. Definitely not meters per second. This is rotations per second. Need to work out how to log this information.

```suggestion
            mVelocity.mut_replace(elevatorMotor1.getVelocity().getValueAsDouble(), MetersPerSecond))));
```
This is still not correct but perhaps closer.